roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -23,26 +27,36 @@
     ceph_stable_rh_storage and
     not ceph_stable_rh_storage_cdn_install and
     not ceph_stable_rh_storage_iso_install
+  tags:
+    - package-install
 
 - name: make sure journal_size configured
   fail:
     msg: "journal_size must be configured. See http://ceph.com/docs/master/rados/configuration/osd-config-ref/"
-  when: journal_size|int == 0
+  when:
+    journal_size|int == 0 and
+    osd_group_name in group_names
```

Comment:
It's not difficult to add those options, it'd just mean requiring post data be sent to an API endpoint that doesn't have anything to do with what the endpoint is actually doing. It feels strange to require ``journal_size`` on an api call that's suppose to only be configuring mons. For reference: https://github.com/ceph/ceph-installer#ceph-installer

When you talk about the handlers triggering restarts, are you referring to the ansible handlers? That shouldn't be an issue for us here because we only operate against one node at a time at these endpoints.

Are there any other issues with having an inconsistent ``ceph.conf`` between monitors and osds?